admin/users/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^$', views.UserFormView.as_view(),
+        name='search'),
+    url(r'^id-(?P<guid>[a-z0-9]+)/$', views.UserView.as_view(),
+        name='user'),
+    url(r'^id-(?P<guid>[a-z0-9]+)/rem-2-factor/$', views.remove_2_factor,
```

Comment:
Consider url of the form `r'^id-(?P<guid>[a-z0-9]+)/two_factor/disable/$'`, or simply `r'^id-(?P<guid>[a-z0-9]+)/two_factor/` that accepts both `POST` and `DELETE` requests. [ADM ticket](https://openscience.atlassian.net/browse/ADM-5) states:

> [After admin removes] User record now has an ADD 2-Factor button (ie: this action can be reversed) (removing two factor deletes the addon, but the user can add it back when they log in.)

This seems to me that if (and only if) it's been disabled through the admin interface, it should be able to be re-enabled? Testing this revealed otherwise.